### DISCUSSION TITLE
Daily Shadow Pattern

### DISCUSSION THREAD

**Discussion by vsoubry** [UMEP-dev/UMEP#247] [Q&A] [answered]
Hello, i was wondering if it is possible to use the Daily Shadow Pattern tool and get the Shadow Fraction for only a period of the day. 
For example i want to know the Shadow Fraction from 12:00 to 20:30. 
There's probably a way to get the same results with other GIS tools and i will find a way if thats not possible, but i just wanted to know, it would make my life easier. 

Thank you!

**Comment by biglimp** [comment, marked as answer]
Hi, Please follow issue #229. As there seem to be more users interested in the same thing I might add that functionallity. I just need to find the time...
